build.gradle

```diff
@@ -1,13 +1,13 @@
 buildscript {
-    ext.kotlin_version = '1.7.22'
+    ext.kotlin_version = '1.6.21'
```

Comment:
curious why are we downgrading kotlin version here ?

Reply:
It seems that there is some sort of conflict between KGP 1.7 and JVM17 that causes R8 to break with a `com.android.tools.r8.internal.Jc: Sealed classes are not supported as program classes` error. It's odd because it runs well on older version (16, 15, 14), I need to spend some time investigating this.